pkg/scheduler/metrics/metrics.go

```diff
@@ -195,11 +195,12 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
-	PreemptionVictims = metrics.NewGauge(
-		&metrics.GaugeOpts{
+	PreemptionVictims = metrics.NewHistogram(
+		&metrics.HistogramOpts{
 			Subsystem:      SchedulerSubsystem,
 			Name:           "pod_preemption_victims",
 			Help:           "Number of selected preemption victims",
+			Buckets:        metrics.LinearBuckets(0, 300, 15),
```

Comment:
How did you choose these values?

Reply:
As stated in the Kubernetes documentation, there can be maximum number of 100 Pods per each server. So if we were to evict three nodes from the cluster, possibly 300 Pods need to be evicted. That is the reason why I have chosen this number. Do you have any suggestion?

Reply:
Shouldn't the victims be the number of victim pods on a single node? Therefore the maximum is 100. And in practice, a node often doesn't contain so many pods. Among victim pods are a subset of all pods on a node. Therefore I assume the distribution will be more towards the lower end, maybe <10.

I suggest something like this: `LinearBuckets(5, 5, 10), start=5, width=5, count=10`.  Justification: we think #victims>50 is pretty rare, therefore [50, +Inf(100)) is considered a single bucket.

Reply:
Thanks. Can you please put a comment there with the justification?

Reply:
Sure!

Reply:
Please resolve this once the comment is added. I will LGTM after that.

Reply:
Sorry, I thought you mentioned that for the tests.